### PR TITLE
Bookmark

### PR DESCRIPTION
**Pull request by annadragan**
Implement bookmark a card function and add character counter

### CODE REVIEW

File: src/components/Card.spec.js [hunk 1]
```diff
@@ -27,17 +27,21 @@ describe('Card', () => {
     expect(explanation).toBeInTheDocument();
   });
 
-  it('calls onClick when clicking the delete icon', () => {
-    const callback = jest.fn();
+  it('calls onClick when clicking the delete or bookmark icon', () => {
+    const bookmarkCard = jest.fn();
+    const deleteCard = jest.fn();
```
Comment (JelenadlS) [inline src/components/Card.spec.js:32]: Looks like a good test🥳

File: src/components/Form.js [hunk 2]
```diff
@@ -1,63 +1,80 @@
-import { useState } from 'react';
+import React, { useState } from 'react';
```
Comment (JelenadlS) [inline src/components/Form.js:1]: im not sure, but i think you dont need to import react itself as it is already part of it 😊

File: src/components/Navigation.spec.js [hunk 3]
```diff
@@ -3,17 +3,14 @@ import { MemoryRouter } from 'react-router-dom';
...
         <Navigation />
       </MemoryRouter>
     );
 
-    const vocabularycardsLink = screen.getByRole('link', {
-      name: /Vocabulary Cards/i,
-    });
-
-    expect(vocabularycardsLink).toBeInTheDocument();
+    const navigation = screen.getByRole('navigation');
```
Comment (JelenadlS) [inline src/components/Navigation.spec.js:13]: I think you should test if the two links are also there and not only the navigation in general or keep the two links out if the it part 😊

**Review by JelenadlS — APPROVED** [review]
I think it looks really really good so far!🤩
I just left small comments for you 😊

File: src/components/Form.js (hunk 2 above)
Comment (annadragan) [reply src/components/Form.js:1]: Hey Jelena, thank you for checking and for your time :)

**Pull request merged**